lib/apis/v2/zz_generated.deepcopy.go

```diff
@@ -19,11 +19,10 @@
 package v2
 
 import (
-	reflect "reflect"
-
 	numorstring "github.com/projectcalico/libcalico-go/lib/numorstring"
 	conversion "k8s.io/apimachinery/pkg/conversion"
 	runtime "k8s.io/apimachinery/pkg/runtime"
+	reflect "reflect"
```

Comment:
this is a weird aliasing, but I'm guessing it's generated code so we can't do anything